spec/factories/resources.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
> Freezing Strings feature improves apps performance by freezing Strings. So, Matz – Ruby’s creator – decided to make all String literals frozen (immutable) by default in Ruby 3.0.

> In order to have a transition path to this coming big change, it was decided to have a magic comment at the beginning of files, so you can use in Ruby 2.3.

https://blog.lucascaton.com.br/2016/01/19/what-is-frozen_string_literal-in-ruby/